waspc/data/Generator/templates/server/src/routes/auth/passport/google/googleDefaults.js

```diff
@@ -0,0 +1,26 @@
+import { upsertUserWithRandomPassword } from '../../../../core/auth.js'
+
+// Default implementation if there is no `auth.methods.google.configFn`.
```

Comment:
Ha reference to Wasp-lang! We avoided that in the rest of the code so far. But we said we will be less careful about that. So ok, if you think it is useful, let's leave it!

Reply:
haha Yeah, I was kinda hesitant to do so, but this file is pretty unique in that I actually link to it from the documentation to show users what the default implementation is, so I think in that context it may be ok. 😅

Reply:
I would keep lower, it doesn't hurt and it communicates intent / documents a bit.